examples/select-table-row-by-cell-content/README.md

```diff
@@ -0,0 +1,27 @@
+# Select a Table Row by Cell Content
+
+**Test Code**: [index.js](index.js)
+
+**Tested Page**: [index.html](index.html)
+
+This example shows how to select a table row based on the cell content.
+
+The tested page includes a table with `Id`, `First name` and `Last name` columns and three rows of data. A click on a row highlights it.
+
+The test identifies the target row by the value of its `Id` column. Then it uses the `t.eql` method to assert that the row has expected cell values and the `t.ok` method to check that it is highlighted.
+
+## TestCafe Functions and Methods Used in This Example
```

Comment:
same as above, pls change to the shorter version

Reply:
edited